com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.3.1] - 2022-12-06
 ### Added
 ### Changed
+- The Editor will stop re-importing Alembic assets when a user changes their project quality settings (HDRP).
```

Comment:
I think this might be a fixed entry.
Deferring to @MaryPerchanok

Reply:
It looks like it reflects the fixed state, but I think it could be phrased better. (change to present tense) 
Suggest"
"When a user changes HDRP quality settings, the Editor no longer re-imports Alembic assets."
(or possibly, "When a user changes HDRP quality settings, the Editor no longer re-imports Alembic assets that already exist in the project.")